### PR TITLE
Add new field description_short to private and public api

### PR DESCRIPTION
**Pull request by iskhakov**
# What this PR does
Required for new Integrations page
Screenshot 2023-04-04 at 20 32 03

## Checklist

- [ ] Unit, integration, and e2e (if applicable) tests updated
- [ ] Documentation added (or `pr:no public docs` PR label added if not required)
- [ ] `CHANGELOG.md` updated (or `pr:no changelog` PR label added if not required)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -20,6 +20,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## v1.2.8 (2023-04-06)
 
+### Added
+
+- Add new field description_short to private api ([#1698](https://github.com/grafana/oncall/pull/1698))
+
```
Comment (joeyorlando) [inline CHANGELOG.md:26]: should be moved to unreleased

**Review by joeyorlando — APPROVED** [review]
LGTM!

File: engine/apps/alerts/models/alert_receive_channel.py [hunk 2]
```diff
@@ -145,6 +145,7 @@ class AlertReceiveChannel(IntegrationOptionsMixin, MaintainableObject):
     smile_code = models.TextField(default=":slightly_smiling_face:")
 
     verbal_name = models.CharField(max_length=150, null=True, default=None)
+    description_short = models.TextField(null=True, default=None)
```
Comment (iskhakov) [inline engine/apps/alerts/models/alert_receive_channel.py:148]: TODO: limit by 250 char

File: CHANGELOG.md (hunk 1 above)
Comment (iskhakov) [reply CHANGELOG.md]: thank you, done

**Pull request merged**